packages/esm-patient-notes-app/src/notes/visit-notes.resource.ts

```diff
@@ -29,21 +28,18 @@ export function useVisitNotes(patientUuid: string): UseVisitNotes {
     'custom:(uuid,display,encounterDatetime,patient,obs,' +
     'encounterProviders:(uuid,display,' +
     'encounterRole:(uuid,display),' +
-    'provider:(uuid,person:(uuid,display)))';
+    'provider:(uuid,person:(uuid,display))),diagnoses';
```

Comment:
Not the main thing, but I'd either put `diagnoses` up at the top after `obs` or on it's own line.